maven-extension/src/main/java/io/opentelemetry/maven/SpanRegistry.java

```diff
@@ -38,6 +42,7 @@ public void setRootSpan(Span rootSpan) {
   }
 
   public Span getSpan(MavenProject mavenProject) {
+    logger.debug("OpenTelemetry: getSpan({}, {})", mavenProject, Thread.currentThread());
```

Comment:
I don't think we should have `Thread.currentThread` in logging messages - most loggers will output the thread info if configured in the pattern anyways.

If we do have it we to check if debug level is enabled before calling the not trivially cheap method

Reply:
Thanks @anuraaga , fixed in PR https://github.com/open-telemetry/opentelemetry-java-contrib/pull/163

> most loggers will output the thread info if configured in the pattern anyways.

Unfortunately it's not the case with the `mvn --debug` / `mvn -X` option but we can remove the thread name as we understood the behaviour of Maven Parallel Builds.